.yamato/build.yml

```diff
@@ -35,7 +35,7 @@ windows:
 
     agent:
       type: Unity::VM
-      image: build-system/unity-extra-windows-10-katana:v0.4.4-699775
+      image: desktop/win10-vs2019-cmake:latest
```

Comment:
The latest tag shouldn't be used in CI, this should specify desktop/win10-vs2019-cmake:v1.0.0-1285066